sensor/kubernetes/complianceoperator/handler_impl.go

```diff
@@ -458,6 +463,156 @@ func (m *handlerImpl) processDeleteScanCfgRequest(request *central.DeleteComplia
 	}
 }
 
+func (m *handlerImpl) handleSyncScanCfgRequest(request *central.SyncComplianceScanConfigRequest) error {
+	go func() {
+		select {
+		case <-m.disabled.Done():
+			log.Errorf("Compliance is disabled. Cannot process request: %s", protoutils.NewWrapper(request))
+			return
+		case <-m.stopSignal.Done():
+			return
+		case <-m.complianceIsReady.Done():
+			if err := m.processSyncScanCfg(request); err != nil {
+				log.Error(err)
+				return
+			}
+		}
+	}()
+	return nil
+}
+
+func generateScanIndex(namespace string, scanName string) string {
+	return fmt.Sprintf("%s-%s", namespace, scanName)
+}
+
+func (m *handlerImpl) getResourcesInCluster(api complianceoperator.APIResource) (map[string]unstructured.Unstructured, error) {
+	cli := m.client.Resource(api.GroupVersionResource())
```

Comment:
This is supposed to be a client/interface to a specific client. I've renamed it to `resourceInterface` to avoid confusion with the `m.client` (dynamic client).